src/threads/thread.c

```diff
@@ -247,9 +237,7 @@ thread_unblock (struct thread *t)
 
   old_level = intr_disable ();
   ASSERT (t->status == THREAD_BLOCKED);
-  /* insert in a sorted priority order */
-  list_sort(&ready_list, priority_desc, NULL);
-  list_insert_ordered (&ready_list, &t->elem, priority_desc, NULL);
+  list_push_back (&ready_list, &t->elem);
```

Comment:
Maybe they're just trying to simplify the threads. No need to have over complicated threads with priority when this project is focused on running user processes???

Reply:
```
/* Inserts ELEM at the end of LIST, so that it becomes the
   back in LIST. */
list_push_back()
```